cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -664,22 +664,27 @@
 
   <property>
     <name>data.tx.prune.plugins</name>
-    <value>data.tx.DefaultHBaseTransactionPruningPlugin</value>
+    <value>data.tx.pruning.plugin</value>
     <description>
       List of transaction pruning plugins; for CDAP HBase tables that use transaction
       functionality to skip or clean invalid data
     </description>
   </property>
 
   <property>
-    <name>data.tx.DefaultHBaseTransactionPruningPlugin.class</name>
+    <name>data.tx.pruning.plugin.class</name>
     <value>co.cask.data2.txprune.DefaultHBaseTransactionPruningPlugin</value>
     <description>
       Class name for the default transaction pruning plugin
     </description>
   </property>
 
   <property>
+    <name>data.tx.prune.state.table</name>
+    <value>${root.namespace}_system:tephra.state</value>
```

Comment:
Is the name 'system' for system namespace configurable? I am assuming its not in this change.